gpu-alloc/src/allocator.rs

```diff
@@ -469,6 +469,25 @@ where
         }
     }
 
+    /// Returns the number of remaining available allocations.
+    ///
+    /// This may be if you need know if the allocator can allocate a number of allocations ahead of time. This
```

Comment:
I guess you've missed a word here `This may be if you need`

Reply:
Yes I have